pkg/csource/options_test.go

```diff
@@ -206,6 +212,11 @@ func enumerateField(OS string, opt Options, field int) []Options {
 			fld.SetString(sandbox)
 			opts = append(opts, opt)
 		}
+	} else if fldName == "SandboxArg" {
+		for _, sandboxArg := range []int64{math.MinInt, math.MaxInt} {
```

Comment:
Lets check for the expected and valid values here.
All the neighbor IFs are checking for the valid ones as I see.
Next, "func (opts Options) Check(OS string) error" checks these invariants for correctness. Lets follow that logic here too and add verification there.

Reply:
We are planning to add more values here and do not know yet how many. I assume that we may have a need to pass special system values or tokens to an API as-is, even though I'm not yet aware of such scenarios, but suspect they may exist.

I mentioned already two scenarios - System account and Restricted account. This number should be multiplied by the number of devices we are planning to support: Pixel 5, 6, 7, 8, etc. So we may want to encode a device name and some settings, for example 71 and 72 settings for pixel 7, 61 and 62 for pixel 6, and so on. For Pixel 8 we may need more information. So it'll be 81, 82 and 811 and 812